docs: add examples to documentation (#2134)
BitGo/BitGoJS: modules/utxo-lib/README.md

- Backed by [BitGo](https://www.bitgo.com/info/)

## Installation

``` bash
# using npm
npm install @bitgo/utxo-lib

# using yarn
yarn add @bitgo/utxo-lib
```

## Setup

JavaScript (ESM)
``` javascript
import * as utxolib from '@bitgo/utxo-lib'
```

NodeJS (CJS)
``` javascript

> const utxolib = require('@bitgo/utxo-lib)'

Reviewer: ```suggestion
const utxolib = require('@bitgo/utxo-lib')
```

|Litecoin|`utxolib.networks.litecoin`|`utxolib.networks.litecoinTest`|
|Zcash|`utxolib.networks.zcash`|`utxolib.networks.zcashTest`|

> > BitGo no longer supports sending funds to [Bitcoin SV](https://blog.bitgo.com/bsv-deprecation-6b3fff4df34c) wallets. Existing customers with said funds will still be able to access and send funds from Bitcoin SV wallets.

Reviewer: ```suggestion
> [Bitcoin SV](https://blog.bitgo.com/bsv-deprecation-6b3fff4df34c) no longer supports sending funds to BitGo wallets. Existing customers with Bitcoin SV in BitGo wallets will still be able to access and sweep funds to an external Bitcoin SV wallet.
```
